lib/noble-device.js

```diff
@@ -71,7 +71,9 @@ NobleDevice.prototype.readDataCharacteristic = function(serviceUuid, characteris
 };
 
 NobleDevice.prototype.writeDataCharacteristic = function(serviceUuid, characteristicUuid, data, callback) {
-  this._characteristics[serviceUuid][characteristicUuid].write(data, false, function(error) {
+  var c = this._characteristics[serviceUuid][characteristicUuid];
+  var withoutResponse = c.properties.indexOf('writeWithoutResponse') !== -1;
```

Comment:
What do you think about changing this line to:

```javascript
var withoutResponse = (c.properties.indexOf('writeWithoutResponse') !== -1) && (c.properties.indexOf('write') === -1) ;
```

to prefer regular writes for characteristics that have both ```write``` and ```writeWithoutResponse``` properties.